src/pages/page-names.constants.ts

```diff
@@ -11,8 +11,10 @@ export const REKEY_SEARCH_PAGE = 'RekeySearchPage';
 export const TEST_RESULTS_SEARCH_PAGE = 'TestResultsSearchPage';
 export const LEGAL_REQUIREMENTS_MODAL = 'LegalRequirementsModal';
 export const REVERSE_DIAGRAM_PAGE = 'ReverseDiagramPage';
+export const REKEY_UPLOAD_OUTCOME_PAGE = 'RekeyUploadOutcomePage';
 
 export const CAT_B = {
+  REKEY_UPLOAD_OUTCOME_PAGE,
```

Comment:
Is there a task to TODO to go back and make sure all previous categories use the Category versions not the direct constant

Reply:
The constant is new since all of then used the same page. This is just a tidy up to make that clear.